docs/compass/03-02-labels.md

```diff
@@ -0,0 +1,155 @@
+# Labels
```

Comment:
An IMO the previous version of this document was simpler :( (even if it was just a proposal for label implementation), I would suggest the following structure of the document:

>Labels allows to enrich top-level objects, like Application, Runtimes with additional information.
>
>// example, setRuntimeLabel, setApplicationLabel
>
>Label can have related Label Definition when a user can define JSON schema for validation of values.
>Label Definitions are optional, but if user use label without creating definition, LabelDefinition without any validation >rules will be created implcitly. 
>
>// example how to create Label Definition, etc
>
>Filtering collections of top-level objects is done only on the label basic.
>User has a possibility to search by label key or by label key and it's value.
>
>// examples
>
>Scenarios label
>
>// description

Also, please ensure that every example is working

Reply:
I guess that we both agree that the document cannot stay in the form of a proposal. Let's discuss how to improve its structure then.